chronosphere/resource_classic_dashboard.go

```diff
@@ -45,6 +47,99 @@ func resourceClassicDashboard() *schema.Resource {
 			StateContext: schema.ImportStatePassthroughContext,
 		},
 		CustomizeDiff: r.ValidateDryRun(&ClassicDashboardDryRunCount),
-		Schema:        tfschema.GrafanaDashboard,
+		Schema:        tfschema.ClassicDashboard,
 	}
 }
+
+// ClassicDashboardDryRunCount tracks how many times dry run is run during validation for testing.
+var ClassicDashboardDryRunCount atomic.Int64
+
+type classicDashboardConverter struct{}
+
+func (classicDashboardConverter) toModel(
+	d *intschema.ClassicDashboard,
+) (*models.Configv1GrafanaDashboard, error) {
```

Comment:
we also have ClassicDashboard in the API, update to use that:
https://docs.chronosphere.io/api-info/definition/operations/ListClassicDashboards